doc/source/whatsnew/v0.25.0.rst

```diff
@@ -20,7 +20,7 @@ Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
 - :meth:`Timestamp.replace` now supports the ``fold`` argument to disambiguate DST transition times (:issue:`25017`)
--
```

Comment:
`indexs` --> `indexes`

Also, reference the issue number.

Reply:
can you be more specific about what has changed, e.g. this now works on multiple column indexes.